oio/cli/clientmanager.py

```diff
@@ -57,6 +58,9 @@ def setup(self):
             self.session = requests.Session()
             self.setup_done = True
             self._admin_mode = self._options.get('admin_mode')
+            # Get the number of meta1 digits from 2 possible sources
```

Comment:
An abusive comment that remained. The second source imagined was the namespace_info. But that kind of configuration is too dangerous too change on the fly.